pyproject.toml

```diff
@@ -35,35 +36,40 @@ check-pylint-import-errors = "hooks.check_pylint_import_errors:main"
 [tool.poetry.dependencies]
 # mandatory dependencies
 python = ">=3.7.2, <4.0"
-mypy = "0.991"
-mypy-extensions = "^0.4.3"
-regex = "^2021.11.10"
+regex = "^2022.1.18"
 requirements-parser = "^0.2.0"
-pylint = "2.15.10"
```

Comment:
I don't think we should pylint yet. The `check-pylint-import-errors` is still in the the `pre-commit-config.yaml` for this repository. Removing pylint means the pre-commit hooks always fail.